### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by mahbubul14**
# Project requirements
 * Create forms to perform the following functions:

   * Creates a Post on behalf of the current_user.
   * Create a comment on behalf of the current_user.
 * Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,4 +8,28 @@ def index
...
+      format.html do
+        if created_post.save
+          redirect_to user_post_path(created_post.author_id, created_post.id), notice: 'Post created !'
+        else
+          render :new, alert: 'Post not created, please try again!'
+        end
+      end
+    end
+  end
+
```
Comment (andersonlebon) [inline app/controllers/posts_controller.rb:29]: - [ ] Kindly note that there is no column in your post table called author_id instead the user is referenced in the post table by the `user_id` column. Kindly fix this by either updating your post table(migration) to reference the user by the `post_id` or user the `user_id in your controllers and views.

image

- [ ] Check your modal also if it has the correct association.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @mahbubul14 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @mahbubul14  👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] You have fixed the schema as suggested by the previous reviewer although In your post.rb file; you have a relationship `belongs_to :author, class_name: 'user'`. this doesn't work because your "user" is in lowercase. please change it to match the name of the class which should be "User", the class_name is supposed to reference exactly the class' name that has this relationship.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — APPROVED** [review]
## Hi, @mahbubul14  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**